tests/ideMode/ideMode005/expected

```diff
@@ -0,0 +1,8 @@
+51c51
+< 0000f6(:return (:ok "   xs : List Nat
+---
+> 0000fa(:return (:ok "   xs : List Nat
+53c53
+< prf2 : map S xs = ?lgjgk" ((8 4 ((:decor :type))) (13 3 ((:decor :type))) (55 3 ((:decor :function))) (59 1 ((:decor :data))) (61 2 ((:decor :bound))) (64 1 ((:decor :keyword))))) 3)
+---
+> prf2 : mapImpl S xs = ?lgjgk" ((8 4 ((:decor :type))) (13 3 ((:decor :type))) (55 7 ((:decor :function))) (63 1 ((:decor :data))) (65 2 ((:decor :bound))) (68 1 ((:decor :keyword))))) 3)
```

Comment:
You should not commit this. You should run the script in the directory to update the test case.